.github/workflows/triage.yml

```diff
@@ -31,7 +31,7 @@ jobs:
             github.event.action != 'closed' && github.event.pull_request.state != 'closed'
         uses: actions/github-script@v3
         with:
-          github-token: ${{ secrets.HOMEBREW_GITHUB_API_TOKEN }}
+          github-token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
With this change we will have GitHub Actions bot approving our PRs instead of BrewTestBot.